src/tmpfiles/tmpfiles.c

```diff
@@ -191,6 +198,30 @@ static int specifier_machine_id_safe(char specifier, void *data, void *userdata,
         return r;
 }
 
+static int specifier_user_name(char specifier, void *data, void *userdata, char **ret) {
+        char *t;
+
+        t = getusername_malloc();
+        if (!t)
+                return -ENOMEM;
+
+        *ret = t;
+        return 0;
+}
+
+static int specifier_user_id(char specifier, void *data, void *userdata, char **ret) {
+
+        if (asprintf(ret, UID_FMT, getuid()) < 0)
+                return -ENOMEM;
+
+        return 0;
+}
+
+static int specifier_user_home(char specifier, void *data, void *userdata, char **ret) {
+
+        return get_home_dir(ret);
+}
```

Comment:
btw, I think this bit should be rebase on top of #7444, which introduces common code for these specifiers, which can be reused here.

Any chance we can get #7444 settled first?